packages/react/src/index.ts

```diff
@@ -56,6 +56,12 @@ export * from "./skeleton";
 export * from "./sprinkles";
 export * from "./styles";
 export * from "./switch";
+export * from "./table";
```

Comment:
Let's move the components to unstable.ts file & remove the web-components for now